composer.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "appwrite/php-runtimes",
+    "description": "A simple Storage library to manage application storage",
+    "type": "library",
+    "keywords": ["php","framework","upf","utopia","storage"],
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Eldad Fux",
+            "email": "eldad@appwrite.io"
+        },
+        {
+            "name": "Torsten Dittmann",
+            "email": "torsten@appwrite.io"
+        }
+    ],
+    "autoload": {
+        "psr-4": {"Appwrite\\Runtimes\\":"src/Runtimes"}
+    },
+    "require": {
+        "php": ">=8.0",
+        "utopia-php/system": "0.4.*"
```

Comment:
Do we still need the system lib as direct dependency for appwrite/appwrite

Reply:
Let me check that 👍🏻

Reply:
Removed 🎉